sdk/storage/azure-storage-file-share/tests/test_file.py

```diff
@@ -859,12 +1012,13 @@ def test_update_range_from_file_url(self, storage_account_name, storage_account_
         # To make sure the range of the file is actually updated
         file_ranges = destination_file_client.get_ranges()
         file_content = destination_file_client.download_file(offset=0, length=512).readall()
-        self.assertEqual(1, len(file_ranges))
-        self.assertEqual(0, file_ranges[0].get('start'))
-        self.assertEqual(511, file_ranges[0].get('end'))
-        self.assertEqual(data, file_content)
+        assert 1 == len(file_ranges)
+        assert 0 == file_ranges[0].get('start')
+        assert 511 == file_ranges[0].get('end')
+        assert data == file_content
 
     @FileSharePreparer()
+    @recorded_by_proxy
     def test_update_range_from_file_url_with_oauth(
             self, storage_account_name, storage_account_key):
```

Comment:
Looks like the multi-line caused the script to miss this one. Should convert this to `kwargs`.